spring-boot-project/spring-boot/src/main/java/org/springframework/boot/orm/jpa/EntityManagerFactoryBuilder.java

```diff
@@ -104,18 +107,34 @@ public void setBootstrapExecutor(AsyncTaskExecutor bootstrapExecutor) {
 		this.bootstrapExecutor = bootstrapExecutor;
 	}
 
+	/**
+	 * Set the PersistenceUnitPostProcessors to be applied to the PersistenceUnitInfo used
+	 * for creating this EntityManagerFactory.
+	 * <p>
+	 * Such post-processors can, for example, register further entity classes and jar
+	 * files, in addition to the metadata read from {@code persistence.xml}.
+	 * <p>
+	 * <b>NOTE: Only applied if no external PersistenceUnitManager specified.</b>
```

Comment:
You wrote that in the PR description and I am not sure I understood what you're trying to convey here. In the context of this particular class, does it matter?

Reply:
I copied from `org.springframework.orm.jpa.persistenceunit.DefaultPersistenceUnitManager#setPersistenceUnitPostProcessors`

Reply:
There's no need to do that.

Reply:
So what should I do here? Remove or simplify theme.

Reply:
Simplify. You can look at other methods in this class for inspiration. More importantly, tests should be added.

Reply:
Okay, I'll try hard.

Reply:
There are not tests related `EntityManagerFactoryBuilder` present, could you provide any suggestions for incoming tests?

Reply:
Maybe I should learn more about `org.springframework.boot.autoconfigure.orm.jpa.AbstractJpaAutoConfigurationTests`.